site/layouts/_default/docs.html

```diff
@@ -31,7 +31,7 @@ <h1 class="bd-title mb-0" id="content">{{ .Title | markdownify }}</h1>
       </div>
 
       {{ if (eq .Page.Params.toc true) }}
-        <div class="bd-toc mt-3 mb-5 my-lg-0 ps-xl-3 mb-lg-5 text-muted">
+        <div class="bd-toc mt-3 mb-5 my-lg-0 ps-xl-3 mb-lg-5 text-body-secondary">
```

Comment:
**[Bootstrap Side]** We might need to extend this class change to the whole doc ?

Reply:
See https://github.com/twbs/bootstrap/pull/37890.